scalability_and_performance/cnf-optimize-data-performance-n3000.adoc

```diff
@@ -0,0 +1,26 @@
+[id="cnf-optimize-data-performance-with-n3000"]
+= Optimizing data plane performance with the Intel® FPGA PAC N3000 and Intel® vRAN Dedicated Accelerator ACC100
+include::modules/common-attributes.adoc[]
+:context: cnf-n3000
+
+toc::[]
+
+include::modules/cnf-intel-n3000-optimize-data-plane.adoc[leveloffset=+1]
+
+include::modules/cnf-installing-the-n3000-operator.adoc[leveloffset=+1]
+
+include::modules/cnf-programming-the-n3000-operator.adoc[leveloffset=+1]
+
+include::modules/cnf-installing-sriov-fec-operator.adoc[leveloffset=+1]
+
+include::modules/cnf-programming-the-sriov-operator.adoc[leveloffset=+2]
+
+include::modules/cnf-programming-the-sriov-fec-operator-acc100.adoc[leveloffset=+2]
+
+include::modules/cnf-verify-sriov-operator-n3000.adoc[leveloffset=+2]
+
+[id="{context}-additional-resources"]
```

Comment:
Team guidelines indicate to set "additional-resources_" as a prefix:

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies